package.json

```diff
@@ -24,6 +24,7 @@
     "lint": "next lint",
     "hps": "local-ssl-proxy --cert ./dev/localhost.pem --key ./dev/localhost-key.pem",
     "https-proxy-start": "local-ssl-proxy --source 3010 --target 3000 --cert ./dev/localhost.pem --key ./dev/localhost-key.pem",
+    "proxy": "local-ssl-proxy --source 3010 --target 3000 --cert ./dev/localhost.pem --key ./dev/localhost-key.pem",
```

Comment:
Tired of typing the whole thing?? Lol.

Reply:
@sherakama lol yeah. I found myself keep using the up arrow 😆